tests/data/config.php

```diff
@@ -16,29 +16,29 @@
 $config = [
     'databases' => [
         'cubrid' => [
-            'dsn' => 'cubrid:dbname=demodb;host=localhost;port=33000',
+            'dsn' => 'cubrid:dbname=demodb;host=cubrid;port=33000',
```

Comment:
@cebe @samdark How should we handle this? Using a separate file for Docker tests. It's theoretically possible to fake this with Docker, but we would loose a lot of  isolation & stability.